composer/workflows/dataflow_template_operator_tutorial/dataflowtemplateoperator_tutorial.py

```diff
@@ -0,0 +1,78 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+bucket_path = bucket_path = "gs://{}".format(bucket_name)
+project_id = models.Variable.get('project_id')
+gce_zone = models.Variable.get('gce-zone')
+
+# [START dataflow_composer_DAG]
+
+import datetime
+from airflow.utils.dates import days_ago
+from airflow import models
+from airflow.contrib.operators.dataflow_operator import DataflowTemplateOperator
+
+#TODO: Replace all instances of project_id with your project id.
+#TODO: Replace bucket_name with your bucket name.
+
+default_args = {
+    #Tell airflow to start one day ago, so that it runs as soon as you upload it
+    "start_date": days_ago(1),
+    'dataflow_default_options': {
+
+        'project': project_id,
+        #Set to your region
```

Comment:
Is this the region of the Composer instance, BQ, or Dataflow? That should be clear from the variable names and/or comments.